R/df_label.R

```diff
@@ -54,10 +54,9 @@ xportr_df_label <- function(.df,
   domain_name <- getOption("xportr.df_domain_name")
   label_name <- getOption("xportr.df_label")
 
-  ## Common section to detect domain from argument or pipes
+  ## Common section to detect domain from argument or attribute
```

Comment:
Other comments also need to be changed. Mind if I commit it here?

Otherwise, this sed command should work:

```bash
sed -i "s/# Common section to detect domain from argument or pipes/# Common section to detect domain from argument or attribute/" R/*.R
```